src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -125,8 +125,8 @@ internal override void OnModelUpdated(Model modelOpt)
             else
             {
                 var selectedItem = modelOpt.SelectedItem;
-                var viewSpan = modelOpt.GetViewBufferSpan(selectedItem.Item.Span);
-                var triggerSpan = modelOpt.GetCurrentSpanInSnapshot(viewSpan, this.TextView.TextSnapshot)
+                var viewSpan = selectedItem == null ? (ViewTextSpan?)null : modelOpt.GetViewBufferSpan(selectedItem.Item.Span);
```

Comment:
I don't think we have any tests for the filtering stuff.  I blame @CyrusNajmabadi.